frontend/packages/dev-console/src/components/pipelines/pipeline-overview/TriggerLastRunButton.tsx

```diff
@@ -1,26 +1,35 @@
 import * as React from 'react';
 import { connect } from 'react-redux';
+import { Button } from '@patternfly/react-core';
 import { impersonateStateToProps } from '@console/internal/reducers/ui';
 import { useAccessReview } from '@console/internal/components/utils';
-import { Button } from '@patternfly/react-core';
+import { AccessReviewResourceAttributes } from '@console/internal/module/k8s';
 import { rerunPipelineAndStay } from '../../../utils/pipeline-actions';
-import { PipelineModel } from '../../../models';
+import { PipelineRunModel } from '../../../models';
 import { usePipelineRunWithUserLabel } from '../../pipelineruns/triggered-by';
 import { getLatestRun, PipelineRun } from '../../../utils/pipeline-augment';
 
 type TriggerLastRunButtonProps = {
   pipelineRuns: PipelineRun[];
+  namespace: string;
   impersonate?;
 };
 
 const TriggerLastRunButton: React.FC<TriggerLastRunButtonProps> = ({
   pipelineRuns,
+  namespace,
   impersonate,
 }) => {
   const latestRun = usePipelineRunWithUserLabel(
     getLatestRun({ data: pipelineRuns }, 'startTimestamp'),
   );
-  const { label, callback, accessReview } = rerunPipelineAndStay(PipelineModel, latestRun);
+  const { label, callback } = rerunPipelineAndStay(PipelineRunModel, latestRun);
+  const accessReview: AccessReviewResourceAttributes = {
+    group: PipelineRunModel.apiGroup,
+    resource: PipelineRunModel.plural,
+    namespace,
+    verb: 'create',
+  };
```

Comment:
Not sure we want to toss out the `accessReview` that the utility provides... perhaps we do this:

```
  const { label, callback, accessReview: utilityAccessReview } = rerunPipelineAndStay(PipelineRunModel, latestRun);
  const defaultAccessReview: AccessReviewResourceAttributes = {
    group: PipelineRunModel.apiGroup,
    resource: PipelineRunModel.plural,
    namespace,
    verb: 'create',
  };

  const accessReview = _.isEmpty(utilityAccessReview) ? defaultAccessReview : utilityAccessReview;
```